src/main/java/reposense/system/LogsManager.java

```diff
@@ -0,0 +1,116 @@
+package reposense.system;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.FileHandler;
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.logging.SimpleFormatter;
+
+/**
+ * Configures and manages the loggers and handlers, including their levels
+ */
+public class LogsManager {
+    private static final int FILE_COUNT = 2;
+    private static final int MEGABYTE = (1 << 20);
+    private static final int MAX_FILE_SIZE_IN_BYTES = 5 * MEGABYTE; // 5MB
+    private static final String LOG_FILE_LOCATION = "./logs/";
+    private static final String LOG_FILE =  LOG_FILE_LOCATION + "reposense.log";
```

Comment:
Add a comment about this block of constants to explain the overall outcome i.e., where files are saved, how and when log files are rolled over, etc.